inst/examples/add.chatty/R/api.R

```diff
@@ -0,0 +1,19 @@
+add <- function(a, b) {
+  logger <- lgr::get_logger("add.chatty")
```

Comment:
Note how users will have to get the logger if they want to use the request ID (and all other things configured by porcelain). They could also pass this through from when the logger is created in the `api`. We could add a function to make this easier.